appveyor.yml

```diff
@@ -27,6 +27,7 @@ install:
   - "pip install nose-timer nibabel nitime"
   - "python setup.py develop"
   - "SET MNE_SKIP_NETWORK_TESTS=1"
+  - "SET MNE_SKIP_FLASH_CALL=1"
```

Comment:
(the test is already decorated with `requires_freesurfer` so will never run on Windows)